docs/backend_readme.md

```diff
@@ -235,4 +235,28 @@ To track the historical events for a certain model based on their id and timesta
 Context.objects.filter(metadata__tracked_model_id=tracked_model_id).order_by("metadata__timestamp")
 ```
 
-</details>
+---
+
+### Troubleshooting
+
+#### Aliases
+
+For a list of command aliases, refer to [.bash_aliases](https://github.com/BetterAngelsLA/monorepo/blob/main/.bash_aliases)
+
+#### Resetting DB
+
+Generally it's best to do this when an error occurs where the local database doesn't accurately reflect the new migration(s).
+
+WARNING: This is a destructive action and the local database will be dropped. As a result, the database will reset with the default seed data.
```

Comment:
related to my comment above - the second sentence here isn't exactly accurate, as it'll reset with seed data only if user runs a second command (per my understanding anyway)